packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -400,7 +412,7 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 				);
 
 				// Flush microtask queue to get PathInfo event logged
-				await delay(0);
+				await clock.tickAsync(0);
```

Comment:
I wouldn't assume this does anything, but if the tests pass maybe it does?  Did you read about it or experiment at all?

Reply:
I think you want `tickAsync`?

```
    /**
     * Advance the clock, firing callbacks if necessary.
     *
     * @param time   How many ticks to advance by.
     * @returns Fake milliseconds since the unix epoch.
     */
    tick: (time: number | string) => number;

    /**
     * Advance the clock, firing callbacks if necessary.
     *
     * Also breaks the event loop, allowing any scheduled promise callbacks to execute _before_ running the timers.
     *
     * @param time   How many ticks to advance by.
     * @returns Fake milliseconds since the unix epoch.
     */
    tickAsync: (time: number | string) => Promise<number>;
```

Reply:
I did not experiment a lot, no. This worked so I just used it.

Reply:
Ha! 🤦‍♂️ That's weird, I must have been blinded from actually reading the code, and just assumed you replaced delay with tick. Sorry for the confusion!